csm/conf/upgrade.py

```diff
@@ -28,5 +30,114 @@ def __init__(self):
 
     async def execute(self, command):
         Log.info("Perform upgrade for csm_setup")
-        # TODO: Implement upgrade logic
+        try:
+            Log.info("Loading Url into conf store.")
```

Comment:
3 Log entries are not required. You could say that Performing upgrade and loading config files

Reply:
Removed extra log entries